packages/cli/src/models/files/ZosPackageFile.js

```
@@ -9,7 +9,25 @@ export default class ZosPackageFile {
 
   constructor(fileName = 'zos.json') {
     this.fileName = fileName
+    let cwd = process.cwd()
+
+    while(!fs.exists(this.fileName)) {
+        if(process.cwd() === "/") {
+            process.chdir(cwd)
+            break
+        }
+
+        try {
+            process.chdir("..")
+        } catch (error) {
+            console.error("couldn't find zos.json")
```

Let's throw instead of just logging